R/build-reference-index.R

```diff
@@ -95,6 +100,24 @@ match_topic <- function(topics, matches) {
     purrr::map_lgl(~ purrr::some(matchers, function(f) any(f(.))))
 }
 
+# Returns the topic file name(s) for a set of aliases (matches)
+topic_files <- function(topics, matches) {
+
+  # Returns the topic file name(s) for a matcher function
+  find_topic_files <- function(matcher) {
+    topics %>%
+      purrr::keep(function(section_topic) any(purrr::map_lgl(section_topic, matcher))) %>%
+      names %>%
```

Comment:
`names()` always returns a vector, so I don't understand why you need an `unlist()` here?